SIT.Manager/ViewModels/PlayPageViewModel.cs

```diff
@@ -13,16 +15,39 @@ public partial class PlayPageViewModel : ObservableRecipient,
                                          IRecipient<ConnectedServerRequestMessage>,
                                          IRecipient<ServerDisconnectMessage>
 {
+    private const string SELECTED_TAB_INDEX_CACHE_KEY = "LastSelectedPlayPageTabIndex";
+
+    private readonly ICachingService _cachingService;
+
     private AkiServer? _connectedServer;
 
     [ObservableProperty]
     private UserControl _playControl;
 
-    public PlayPageViewModel()
+    [ObservableProperty]
+    private int _selectedTabIndex = 0;
+
+    public PlayPageViewModel(ICachingService cachingService)
     {
+        _cachingService = cachingService;
+
+        if (_cachingService.OnDisk.TryGet(SELECTED_TAB_INDEX_CACHE_KEY, out CacheValue<int> indexValue))
+        {
+            SelectedTabIndex = indexValue.Value;
+        }
+
         PlayControl = new ServerSelectionView();
     }
 
+    partial void OnSelectedTabIndexChanged(int value)
+    {
+        if (_cachingService.OnDisk.Exists(SELECTED_TAB_INDEX_CACHE_KEY))
+        {
+            _cachingService.OnDisk.Remove(SELECTED_TAB_INDEX_CACHE_KEY);
+        }
+        _cachingService.OnDisk.Add(SELECTED_TAB_INDEX_CACHE_KEY, value);
+    }
+
```

Comment:
Sorry about this. I'll add a TryRemove soon. This could cause issues in multithreaded situations

Reply:
An update value function would have worked well here too I think, if that's not too much effort.